tests/adobe/mixins/test_order.py

```
@@ -322,3 +324,88 @@ def test_get_preview_order_line_item(
+
+    assert flex_discounts == {}
+
+
+def test_get_flex_discounts_per_base_offer_error(
+    adobe_client_factory,
+    requests_mocker,
+    settings,
+    order_preview_discounts_resp_factory,
+    preview_discounts_payload_factory,
+    mock_order,
+    flex_discounts_factory,
```

Ruff (0.14.6)

Unused function argument: `flex_discounts_factory`

(ARG001)